### PR TITLE
[SOL-2083] Check if user is enrolled in course only if the request arrives from the Offer Landing Page

### PR DESCRIPTION
**Pull request by marjev**
@mjfrey Here's a PR for [SOL-2083](https://openedx.atlassian.net/browse/SOL-2083).

Please take a look.

### CODE REVIEW

**Comment by openedx-webhooks** [comment]
Thanks for the pull request, @marjev! It looks like you're a member of a company that does contract work for edX. If you're doing this work as part of a paid contract with edX, you should talk to edX about who will review this pull request. If this work is not part of a paid contract with edX, then you should ensure that there is an OSPR issue to track this work in [JIRA](https://openedx.atlassian.net), so that we don't lose track of your pull request.

To automatically create an OSPR issue for this pull request, just visit this link: https://openedx-webhooks.herokuapp.com/github/process_pr?repo=edx%2Fecommerce&number=953

File: ecommerce/extensions/basket/views.py [hunk 1]
```diff
@@ -54,7 +54,10 @@ def get(self, request):
 
         # If the product is not an Enrollment Code, we check to see if the user is already
         # enrolled to prevent double-enrollment and/or accidental coupon usage
-        if product.get_product_class().name != ENROLLMENT_CODE_PRODUCT_CLASS_NAME:
+        print request.GET
```
Comment (mjfrey) [inline ecommerce/extensions/basket/views.py]: remove the prints

Comment (marjev) [reply ecommerce/extensions/basket/views.py]: Prints removed.

File: ecommerce/extensions/basket/views.py [hunk 2]
```diff
@@ -54,7 +54,7 @@ def get(self, request):
 
         # If the product is not an Enrollment Code, we check to see if the user is already
         # enrolled to prevent double-enrollment and/or accidental coupon usage
```
Comment (mjfrey) [inline ecommerce/extensions/basket/views.py]: Please update this comment to say we will only check if this is a coupon redemption.

Comment (marjev) [reply ecommerce/extensions/basket/views.py]: I've updated the comment. Please take a look to see if its meaning is understandable enough.

**Comment by mjfrey** [comment]
👍

**Pull request merged**